Add instructions for filtering monitors that do not contain a tag. (#7175)
DataDog/documentation: content/en/tagging/using_tags.md

{{< tabs >}}
{{% tab "Manage Monitors" %}}

> To filter monitors by [assigned tags][1], use the search bar or facet checkboxes. The search bar format is `tag:<KEY>:<VALUE>`, for example `tag:service:coffee-house`. You can also filter monitors that do not have a tag, for example `tag:-service:coffee-house`. **Note**: monitor tags are different and separate from metric tags.

Reviewer: ```suggestion
To filter monitors by [assigned tags][1], use the search bar or facet checkboxes. The search bar format is `tag:<KEY>:<VALUE>`, for example `tag:service:coffee-house`. To exclude monitors with a specific tag from your search, use `-`, for example `tag:-service:coffee-house`. **Note**: Monitor tags are different and separate from metric tags.
```
